packages/nextcloud/lib/src/client.dart

```diff
@@ -1,43 +1,33 @@
 import 'package:dynamite_runtime/http_client.dart';
 import 'package:universal_io/io.dart';
 
-/// Different app types to register for
-enum AppType {
-  /// Will only receive Talk notifications
-  talk('Mozilla/5.0 (Android) Nextcloud-Talk'),
-
-  /// Will receive all notifications except Talk notifications if another Talk
-  /// app is already registered for the user
-  nextcloud('Mozilla/5.0 (Android) Nextcloud-android'),
-
-  /// Default. Same problem with notifications as the [nextcloud] type
-  unknown(null);
-
-  // ignore: public_member_api_docs
-  const AppType(this.userAgent);
-
-  // ignore: public_member_api_docs
-  final String? userAgent;
-}
-
-// ignore: public_member_api_docs
+/// A client configuring the clients for all Nextcloud APIs.
+///
+/// To access the APIs of a particular app import the extensions through `package:nextcloud/{id}.dart`.
 class NextcloudClient extends DynamiteClient {
-  // ignore: public_member_api_docs
+  /// Creates a new Nextcloud API client.
+  ///
+  /// [baseURL] has to point to the Nextcloud server webroot.
+  /// If the server does not support clean URLs `/index.php` has to be appended to the URL.
```

Comment:
Sure? All our request already include the `/index.php/` in the url.

Reply:
Ah true I forgot about that. I thought we were adding it when instantiating the client...